test/runtests.jl

```diff
@@ -59,10 +59,10 @@ end
     C2 = Contractor(vars, y+z)
     @test C2(A,X) == IntervalBox(0.5..1.5, 0.5..0.5, 0.5..0.5)
 
-    C1 = Contractor([:x,:y,:z], x+y)
+    C1 = Contractor([x,y,z], x+y)
     @test C1(A,X) == IntervalBox(0.5..0.5, 0.5..0.5, 0.5..1.5)
 
-    C2 = Contractor([:x,:y,:z], y+z)
+    C2 = Contractor([x,y,z], y+z)
     @test C2(A,X) == IntervalBox(0.5..1.5, 0.5..0.5, 0.5..0.5)
```

Comment:
You need to make this result be different from the previous one in order to check that `x + y` and `y + z` actually give different results.

Reply:
It is already shown that results are different for `x+y` and `y+z`.

Reply:
C1(A,X) and C2(A,X) are giving different results

Reply:
You're right, sorry -- my mistake.